mzLib/Test/TestRetentionTimePrediction.cs

```diff
@@ -57,543 +58,543 @@ public TestRetentionTimePrediction()
-                {"VEDGLHIISPELQEEEEQSHSQR", 28.77},
-                {"TIDPNGLHLPSYSPSPQLIFIIQGK", 45.07},
-                {"GGQQQEEESEEQNEGNSVLSGFNVEFLAHSLNTK", 37.57},
-                {"RGGQQQEEESEEQNEGNSVLSGFNVEFLAHSLNTK", 36.99},
-                {"ALEAFK", 16.38},
-                {"TFLWGR", 26.93},
-                {"NEPWWPK", 25.98},
-                {"LLYPHYR", 22.29},
-                {"SDYVYLPR", 25.01},
-                {"EEELNNLR", 15.37},
-                {"GSAEFEELVK", 26.15},
-                {"SSDFLTYGLK", 29.89},
-                {"ELVEVGHGDKK", 14.09},
-                {"DNPNWTSDKR", 11.67},
-                {"HASDELYLGER", 21.11},
-                {"LPTNILSQISPLPVLK", 43.3},
-                {"NWVFTEQALPADLIK", 40.97},
-                {"FQTLIDLSVIEILSR", 56.36},
-                {"EHLEPNLEGLTVEEAIQNK", 36.57},
-                {"ATFLEGIISSLPTLGAGQSAFK", 52.05},
-                {"IFFANQTYLPSETPAPLVHYR", 43.17},
-                {"IYDYDVYNDLGNPDSGENHARPVLGGSETYPYPR", 36.67},
-                {"SQIVR", 8.97},
-                {"VEGGLR", 8.67},
-                {"SEFDR", 7.5},
-                {"HSYPVGR", 10.87},
-                {"EQSHSHSHR", -0.82},
-                {"TANSLTLPVLR", 29.66},
-                {"AAVSHVQQVLR", 23.22},
-                {"ENIADAAGADLYNPR", 27.31},
-                {"EEEEEEEEDEEKQR", 5.84},
-                {"IRENIADAAGADLYNPR", 28.95},
-                {"VESEAGLTETWNPNNPELK", 31.91},
-                {"VFYLGGNPETEFPETQEEQQGR", 32.3},
-                {"TIDPNGLHLPSFSPSPQLIFIIQGK", 48.01},
-                {"GQLVVVPQNFVVAEQAGEEEGLEYVVFK", 48.85},
-                {"KGQLVVVPQNFVVAEQAGEEEGLEYVVFK", 47.37},
-                {"LLENQK", 8.32},
-                {"QIEELSK", 12.03},
-                {"NQVQSYK", 6.05},
-                {"FFEITPK", 25.11},
-                {"NENQQGLR", 6.3},
-                {"KQIEELSK", 13.2},
-                {"ILLEEHEK", 18.62},
-                {"EEDDEEEEQR", 4.04},
-                {"DLTFPGSAQEVDR", 24.13},
-                {"QSYFANAQPQQR", 15.52},
-                {"ILLEEHEKETHHR", 17.28},
-                {"NFLAGEEDNVISQIQK", 32.48},
-                {"LTPGDVFVIPAGHPVAVR", 37.28},
-                {"EEDDEEEEQREEETK", 5.89},
-                {"ASSNLNLLGFGINAENNQR", 35.42},
-                {"NPQLQDLDIFVNYVEIK", 46.41},
-                {"KNPQLQDLDIFVNYVEIK", 45.53},
-                {"NENQQGLREEDDEEEEQR", 10.37},
-                {"GDQYAR", 3.5},
-                {"GDYYAR", 7.6},
-                {"EVYLFK", 24.15},
-                {"GKEVYLFK", 25.17},
-                {"VLYGPTPVR", 23.15},
-                {"TGYINAAFR", 23.93},
-                {"TNEVYFFK", 28.18},
-                {"TLDYWPSLR", 32.85},
-                {"KTLDYWPSLR", 32.13},
-                {"VLYGPTPVRDGFK", 27.02},
-                {"YVLLDYAPGTSNDK", 31.2},
-                {"SSQNNEAYLFINDK", 26.36},
-                {"NTIFESGTDAAFASHK", 26.97},
+                { "LVEYR", 10.69 },
```

Comment:
I don't see any tests for modified peptides?
Tests for what happens when modifications are not supported